irohad/main/server_runner.cpp

```diff
@@ -43,10 +40,16 @@ void ServerRunner::run() {
 
   console.info("Server listening on {}", serverAddress_);
```

Comment:
Sorry, actually I think `console` is logging service because I uses `logger::Logger`. But naming is problem.
I'll rename `console` -> `Log`. (I wanted to use `log` at first but this is std function...)